code/modules/admin/verbs/debug.dm

```diff
@@ -468,10 +468,12 @@
 	set name = "Spawn Material Stack"
 	if(!check_rights(R_DEBUG)) return
 
-	var/decl/material/material = input("Select material to spawn") as null|anything in SSmaterials.materials
```

Comment:
This is a regression - `SSmaterials.materials` is an instance list that uses the `name` var.
I would suggest adding a sort to the end of `SSmaterials` init rather than sorting the entire list of several hundred materials every time someone uses the verb.

Reply:
Not sure what you mean there about it being "an instance list that use the name var"? Its not associative, so there's no name?
And I guess I could sort it after material init. But do we know if anything creates material decl on the fly?

Reply:
You can't create material decls on the fly by definition.
The code you are replacing does not use the assoc list.

Reply:
True.